packages/rlc-codegen/src/generateSchemaType.ts

```diff
@@ -0,0 +1,53 @@
+import { ObjectDestructuringAssignment, Project } from "ts-morph";
```

Comment:
Could you add license term for each of the files?

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
updated.